packages/react-error-overlay/package.json

```diff
@@ -50,6 +51,7 @@
     "eslint-plugin-import": "2.20.1",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-react": "7.19.0",
+    "eslint-plugin-react-hooks": "^3.0.0",
```

Comment:
This seems like a weird addition to a package we won't publish...

Reply:
Yeah it was causing issues in building that it was required but couldn't be found so publishing was blocked, the issue was happening in upstream too so not sure how they are still publishing CRA